buildbot-host/buildmaster/master.cfg

```diff
@@ -504,10 +505,15 @@ for factory_name, factory in factories.items():
 # We need to create schedulers after the builders, because otherwise the build
 # name lists are not available yet.
 c['schedulers'] = []
+
+# Only build if any of the files in the Change match this regular expression
+openvpn_filter_fn = r'^(src/(openvpn)/(\w|-)+\.(c|h)|m4/(\w|-)+\.m4|compat.m4|configure.ac|Makefile.am|version.m4)$'
```

Comment:
That regex definitely needs more work. Some initial ideas:

- Needs to cover all relevant Makefile.am files, not just top-level
- Needs to cover tests/
- Needs to cover src/compat/
- Needs to cover include/
- Needs to cover config.h.*

Reply:
Asking about this in tomorrow's community meeting would probably be quickest. I won't make it, but "what modifications should trigger a build" is a fairly straightforward question I think :smile:. So, I'll add it to the topic list.